sdk/identity/azure-identity/azure/identity/_internal/shared_token_cache.py

```diff
@@ -229,6 +229,11 @@ def _get_refresh_tokens(self, account):
             message = "Error accessing cached data: {}".format(ex)
             six.raise_from(CredentialUnavailableError(message=message), ex)
 
+    @property
+    def token_refresh_offset(self):
+        # type: (None) -> int
+        return DEFAULT_REFRESH_OFFSET
```

Comment:
This is incorrect because there's effectively a hardcoded 5 minute offset (see line 211). Do you plan to add the feature to these credentials?

Reply:
OK. I will update it to 300s.